test/dialect/test_sqlite.py

```diff
@@ -4,6 +4,7 @@
 import datetime
 import json
 import os
+from unittest.mock import ANY
```

Comment:
**mike bayer** (zzzeek) wrote:

for backporting this has to be "from testing.mock import ANY"

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4200